react-native/react/images/folders/fa-folder-private-empty.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think this private-empty is the same as the font-awesome one, but the other svgs are different. Having this one as an svg makes the positioning consistent across the images.